pkg/meta/model/job.go

```diff
@@ -111,7 +111,7 @@ const (
 	ActionDropResourceGroup      ActionType = 70
 	ActionAlterTablePartitioning ActionType = 71
 	ActionRemovePartitioning     ActionType = 72
-	ActionAddVectorIndex         ActionType = 73
+	ActionAddColumnarIndex       ActionType = 73
```

Comment:
This is usually handled by adding a new type, such as https://github.com/pingcap/tidb/pull/60244/files#diff-c51ad6b394f865592721a3e18d58574176394570e3efcff7703e8bd0c3c5c2efL74

Reply:
Since both vector index and inverted index are columnar indexes, and we are expected to add more columnar indexes later, we are trying to make support of new columnar indexes as simple as possible. The main logic of columnar indexes in DDL is the same, so maybe using the same `Action` will make it simpler and it would not break the compatibility.